ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -421,6 +485,135 @@ const CreateRunPage = () => {
               )}
             />
 
+            <h3 className='mt-4'>Parameters</h3>
+            <div className='text-sm text-gray-500'>
+              A map with custom parameters for the whole ORT run. The parameters
+              can be evaluated by the validation script executed by the config
+              worker. The script can convert these parameters to specific job
+              configurations understood by ORT Server. The first column is the
+              key and the second is the value.
+            </div>
+            {pFields.map((field, index) => (
+              <div
+                key={field.id}
+                className='my-2 flex flex-row items-start space-x-2'
+              >
+                <div className='flex-auto'>
+                  <FormField
+                    control={form.control}
+                    name={`jobConfigs.parameters.${index}.key`}
+                    render={({ field }) => (
+                      <FormItem>
+                        <FormControl>
+                          <Input {...field} />
+                        </FormControl>
+                        <FormMessage />
+                      </FormItem>
+                    )}
+                  />
+                </div>
+                <div className='flex-auto'>
+                  <FormField
+                    control={form.control}
+                    name={`jobConfigs.parameters.${index}.value`}
+                    render={({ field }) => (
+                      <FormItem>
+                        <FormControl>
+                          <Input {...field} />
+                        </FormControl>
+                        <FormMessage />
+                      </FormItem>
+                    )}
+                  />
+                </div>
+                <Button
+                  type='button'
+                  variant='outline'
+                  size='sm'
+                  onClick={() => {
+                    pRemove(index);
+                  }}
+                >
+                  <TrashIcon className='h-4 w-4' />
+                </Button>
+              </div>
+            ))}
+            <Button
+              size='sm'
+              className='mt-2'
+              variant='outline'
+              type='button'
+              onClick={() => {
+                pAppend({ key: '', value: '' });
+              }}
+            >
+              Add parameter
+              <PlusIcon className='ml-1 h-4 w-4' />
+            </Button>
+
+            <h3 className='mt-4'>Labels</h3>
+            <div className='text-sm text-gray-500'>
+              The labels of this run. The first column is the key and the second
```

Comment:
Labels are strictly speaking not attached to runs, but to results (ORT result files). Again, I'd prefer @mnonnenmacher to share his view for a good wording. Also here, instead of the last sentence, I'd prefer column headers.

Reply:
In the server context it's fine to say labels are associated with a run, the result files are merely a report format here.